libsolidity/analysis/ReferencesResolver.cpp

```diff
@@ -101,7 +93,11 @@ bool ReferencesResolver::visit(Identifier const& _identifier)
 		string suggestions = m_resolver.similarNameSuggestions(_identifier.name());
 		string errorMessage =
 			"Undeclared identifier." +
-			(suggestions.empty()? "": " Did you mean " + std::move(suggestions) + "?");
```

Comment:
This gets rather complicated with the ternary operators. Perhaps it is better to construct `errorMessage` in multiple statements and use `if` instead?

Reply:
Also spacing: Exactly one space before and after `?` and `:` and usually we do not use additional parentheses.